src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationCollectionBindingTests.cs

```diff
@@ -1417,275 +1422,739 @@ public void TestCanBindListPropertyWithoutSetter()
             Assert.Equal(new[] { "a", "b" }, options.ListPropertyWithoutSetter);
         }
 
-        private class UninitializedCollectionsOptions
```

Comment:
>UninitializedCollectionsOptions [](http://example.com/codeflow?start=22&length=31)

Why you removed such classes? Testing classes has multiple types is a good test cases to have instead of testing each type separately.

Reply:
They were not removed, just moved to separate `TestClasses` files. This made testing easier when I was iterating on getting successful compilations.